content/blog/posts/o3de-user-defined_properties-2022-03.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "O3DE User Defined Properties"
+date: 2022-03-08
+slug: blog-udp
+author: Allen Jackson
+blog_img: "/images/blog/udp-intro/editor_def_mat_port.png"
+full_img: "/images/blog/udp-intro/editor_def_mat_port.png"
+---
+
+# O3DE User Defined Properties
+
+Artists store metadata in source scenes (e.g. blend, ma, 3ds file extensions) to user defined properties (UDP) on scene nodes. This user defined property metadata can be exported into source scene asset files (e.g. FBX or glTF). Game teams use this UDP metadata to tag mesh data for engine purposes such as splitting up meshes into level of details (LODs), flagging a simplified mesh as a collision mesh to use as a trigger or hit detection, and assigning scene features like render materials to engine models.
+
+Our team has added the ability for the O3DE asset pipeline to read in this UDP metadata so that scene building logic can act using the UDP metadata. This allows the artists to continue to work primarily in the tools they have mastered such as Maya or Blender. For example, since the UDP metadata can assign a material to a mesh node the artist will not need to open the O3DE Editor in order to assign the correct material the exported model. 
```

Comment:
nit: External users do not know what "Our team" referes to here, consider just using "O3DE".

"For example, since the UDP metadata can assign a material to a mesh node[,] the artist will not.."